server/api_design/validate_contracts.py

```
@@ -0,0 +1,80 @@
+                    json.dumps(spec["example"])
+                else:
+                    # Try auto-locate
+                    model = endpoint.get("model")
+                    verb = endpoint.get("method").lower()
+                    example_path = os.path.join(EXAMPLES_DIR, model, f"{verb}.{status_code}.json") if model else None
+                    if not example_path or not os.path.exists(example_path):
+                        print(f"  ⚠️ No example or $ref for {endpoint['method']} {endpoint['endpoint']} [{status_code}]")
+                        continue
+                # If we reach here, the example exists and is valid
+                print(f"  ✅ {endpoint['method']} {endpoint['endpoint']} [{status_code}]")
+            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)